core/src/main/kotlin/com/gitlab/kordlib/core/behavior/MemberBehavior.kt

```diff
@@ -71,10 +71,19 @@ interface MemberBehavior : Entity, UserBehavior {
      *
      * @throws [RestRequestException] if something went wrong during the request.
      */
-    suspend fun addRole(roleId: Snowflake) {
+    suspend fun addRole(roleId: Snowflake) {//TODO remove in Kord 0.7.0
```

Comment:
Add Depreciated annotation?

Reply:
Good point, I forgot about the hidden deprecation